qlib/workflow/task/utils.py

```diff
@@ -67,22 +79,23 @@ def align_time(self, time_point, tp_type="start"):
         """
         return self.cals[self.align_idx(time_point, tp_type=tp_type)]
 
-    def align_seg(self, segment):
+    def align_seg(self, segment: Union[dict, tuple]):
         if isinstance(segment, dict):
             return {k: self.align_seg(seg) for k, seg in segment.items()}
         elif isinstance(segment, tuple):
             return self.align_time(segment[0], tp_type="start"), self.align_time(segment[1], tp_type="end")
         else:
             raise NotImplementedError(f"This type of input is not supported")
 
-    def truncate(self, segment, test_start, days: int):
+    def truncate(self, segment: tuple, test_start, days: int):
         """
         truncate the segment based on the test_start date
 
         Parameters
         ----------
-        segment :
+        segment : tuple
             time segment
+        test_start
         days : int
             The trading days to be truncated
             大部分情况是因为这个时间段的数据(一般是特征)会用到 `days` 天的数据
```

Comment:
Try `rg '[p{Han}]'`

Reply:
sorry, I can't understand what is `rg '[p{Han}]'`